### ISSUE TITLE
Transfer ownership to Sugar Labs

### ISSUE THREAD

**Issue by chimosky** [shaansubbaiah/swiftfeet-activity#1]
@shaansubbaiah would you transfer ownership of this repo to Sugar Labs, seeing as there's not a lot of work going on here, I doubt there's a reason to keep two separate repos.

**Comment by shaansubbaiah** [comment]
Hi @chimosky , sorry for replying so late, not sure how I missed this.

Sure, I see no reason not to. I was unable to transfer the repo due to the error "sugarlabs/swiftfeet-activity already exists and You don’t have the permission to create public repositories on sugarlabs". I believe the fork will have to be deleted/renamed and I will need to temporarily get access to create repos in the sugarlabs org. IIRC the reason we had created the repo in my account first and then forked it was due to lack of permissions.

Do to these reasons, would it be better to just maintain the forked repo in the sugarlabs org?
